### ISSUE TITLE
Allow custom_settings of spider to be edited via UI

### ISSUE THREAD

**Issue by botzill** [my8100/scrapydweb#90]
This is not a bug clearly, just a suggestion.

I was wondering if it makes sense to allow `custom_settings` defined per spider to be edited via UI? Most of the time we have had multiple spiders and each has its own settings and would be nice to have such feature where we edit these settings on the fly. 

So far this project is the best!

p.s Maybe allow options to submit a feature request rather then bug?

**Comment by my8100** [comment]
You can custom the `SCHEDULE_ADDITIONAL` option to override the `custom_settings` of your spiders. 

https://github.com/my8100/scrapydweb/blob/8104386438cb7e18e5b619c53aedf22dc5bb8954/scrapydweb/default_settings.py#L150-154

Labels like Suggestion and Feature request are also available when opening an issue.

**Comment by botzill** [comment]
Yes, that settings is great. But when we have multiple spiders each having it's own settings we can't use `SCHEDULE_ADDITIONAL` as this is general settings for all. I was thinking of a way to set this settings when we select a specific spider, to load it's `custom_settings` and be able to set them.

**Comment by my8100** [comment]
You can set the SCHEDULE_ADDITIONAL option like this and update it manually after selecting a spider. 
```
SCHEDULE_ADDITIONAL = "-d arg=value1\r\n-d arg=value2" 
```
